templates/demo/source/docinfo.ptx

```diff
@@ -4,6 +4,29 @@
 <!-- This is where you can define macros and other book-wide -->
 <!--  settings.                                              -->
 <docinfo xmlns:xi="http://www.w3.org/2001/XInclude">
+  <!--
```

Comment:
This is very nice.  I don't know whether it is true that document-id is only used for Runestone though.  @rbeezer , is `document-id` still needed for the catalog and other things?

Reply:
> is document-id still needed

It was used in the catalog as a way to "claim" one, and ensure global uniqueness as a result.

It might still be used to avoid clashes between books (such as in situations similar to Runestone).